test/e2e/app-dir/use-cache-handler-alias/use-cache-handler-alias.test.ts

```diff
@@ -0,0 +1,64 @@
+import { nextTestSetup } from 'e2e-utils'
+import { retry } from 'next-test-utils'
+
+describe('use-cache-handler-alias', () => {
+  const { next, skipped } = nextTestSetup({
+    files: __dirname,
+    // Skip deployment so we can test the custom cache handlers log output
+    skipDeployment: true,
+  })
+
+  if (skipped) return
+
+  it('should use cache handler alias if provided', async () => {
```

Comment:
I think it's fine to simplify this test to only assert on the expected logs from the custom cache handler. The caching logic itself is just delegated to the default cache handler, which is already tested in other test suites, so we don't need to do that here as well.